pom.xml

```diff
@@ -170,7 +170,7 @@
         <dependency>
             <groupId>net.masterthought</groupId>
             <artifactId>cucumber-reporting</artifactId>
-            <version>3.15.0</version>
+            <version>3.16.0</version>
```

Comment:
are you working on the most recent master version ? 
https://github.com/damianszczepanik/cucumber-sandwich/blob/master/pom.xml#L173

Reply:
Yeah 3.16 is the most recent edition.
https://github.com/damianszczepanik/cucumber-reporting/releases

Reply:
So why did you need to upgrade from older version 3.15?